packages/utils/modify-config-helper.ts

```diff
@@ -87,10 +87,11 @@ export default function modifyHelperUtil(
 			}
 		};
 	}
+	
 	env.registerStub(generator, generatorName);
-
-	env.run(generatorName)
-		.then(
+	env.run(generatorName, {
+		configFile
```

Comment:
What's this change for?

Reply:
To pass the config file name to the generator.

Reply:
This change is probably ok and needed as we need this to be dynamic. Just gotta check that it works. Change is that the generator will have output of the config file supplied vs webpack.config.js

Reply:
yes, I used this in my machine and its working fine. This can use workable with the other generators too like the `update` one and in `add` too